prod/westeurope/internal/api/apim/api_management_api_public_r3/swagger.json.tmpl

```diff
@@ -0,0 +1,57 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "IO API Public",
+    "version": "1.0",
+    "description": "IO API Public."
+  },
+  "host": "${host}",
+  "basePath": "/public",
```

Comment:
@gunzip which path should I use here?

Reply:
why not `/${path}` ? (does this get replaced by the values into tfvars ?)

see https://github.com/pagopa/io-infrastructure-live/blob/master/dev/westeurope/api_management_apis/io-public/swagger.json#L16

Reply:
absolutely, you are right